resources/assets/js/countdown.js

```diff
@@ -67,14 +69,13 @@ function formatFromNow(timestamp) {
 /**
  * Initialises all countdown fields on the page.
  */
-$(function () {
-  $.each($('[data-countdown-ts]'), function (i, e) {
-    const span = $(e);
-    const timestamp = span.data('countdown-ts');
-    const text = span.html();
-    span.html(text.replace('%c', formatFromNow(timestamp)));
+ready(function () {
```

Comment:
arrow-function?

Reply:
Mention this one time per PR is sufficient :wink: